### PR TITLE
i2c 0.1.2 (and dependency fix for 0.1.1)

### PR DESCRIPTION
**Pull request by mwweissmann**
fixes the build issues from cleaning up package _stdint_

### CODE REVIEW

File: packages/i2c/i2c.0.1.1-0/opam [hunk 1]
```diff
@@ -18,8 +18,8 @@ build-doc: [ "ocaml" "setup.ml" "-doc" ]
 depends: [
   "result"
   "base-unix"
-  "stdint"
+  "stdint" {<= "0.3.0"}
   "ocamlfind" {>= "1.5"}
   "ocamlbuild" {build}
 ]
-available: [ os = "linux" ]
+available: [ os = "linux" & os != "alpine" ]
```
Comment (yallop) [inline packages/i2c/i2c.0.1.1-0/opam]: The test against `"alpine"` won't work, unfortunately.  See the discussion here: https://github.com/ocaml/opam-repository/pull/9915#issuecomment-320282598

Comment (mwweissmann) [inline packages/i2c/i2c.0.1.1-0/opam]: I see; I added a _conf-package_ to address the real dependency (linux kernel header files being available)

**Comment by mwweissmann** [comment]
might I have a restart of the buildbot? It cannot find the newly added conf-linux-libc-dev package

**Comment by yallop** [comment]
opam can't find the new package because the directory structure isn't quite what it expects.  Could you add a version number to the subdirectory, e.g.

```
packages/conf-linux-libc-dev/conf-linux-libc-dev.0/ 
```
?

**Comment by mwweissmann** [comment]
I might need a hint what _camelus_ doesn't like particularly about _conf-linux-libc-dev.0_

**Comment by yallop** [comment]
Changing the `opam-version` field to
```
opam-version: "1.2"
```
and adding the lines

```
bug-reports: "https://github.com/ocaml/opam-repository/issues"
dev-repo: "https://github.com/ocaml/opam-repository.git"
```
should make camelus happy.

**Comment by yallop** [comment]
I've pushed a commit (80d31e88) with a typo fix; when CI finishes I think this can be merged.

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Pull request merged**